directory_sso_api_client/user.py

```diff
@@ -5,6 +5,7 @@ class UserAPIClient(BaseAPIClient):
 
     endpoints = {
         'session_user': 'session-user/',
+        'bearer_user': 'oauth2/user-profile/v1/',
```

Comment:
`bearer token` is created is used by the oauth2 consumer. Endpoint name should be sth like `oauth2_user_profile`